### ISSUE TITLE
disk.frame doesn't work on fst written by write_fst

### ISSUE THREAD

**Issue by kendonB** [xiaodaigh/disk.frame#138]
The below examples seems so simple yet fails. There must be something I'm doing wrong...

```` r
library(fst)
library(disk.frame)
path <- paste0(tempfile(pattern = "folder"))
dir.create(path)
write_fst(iris, file.path(path, "iris.fst"))
list.files(path)
#> [1] "iris.fst"
df = disk.frame::disk.frame(path)
df
#> path: "/tmp/RtmpUzANvL/folder7f68686c046f"
#> nchunks: 1
#> nrow: 150
#> ncol: 5
df %>% 
  collect()
#> Warning in collect.disk.frame(.): NAs introduced by coercion
#> Warning in get_chunk.disk.frame(x, .x, full.names = TRUE): The chunk /tmp/
#> RtmpUzANvL/folder7f68686c046f/NA.fst does not exist; returning an empty
#> data.table
#> Null data.table (0 rows and 0 cols)
````

<sup>Created on 2019-09-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by xiaodaigh** [comment]
Ok @kendonB, for now I have disabled arbitrary names for fst files. You need to number them starting from "1.fst". Or use the `add_chunk(data.frame)` function. 
```r
library(fst)
library(disk.frame)
setup_disk.frame() # for multi worker processing 

path <- paste0(tempfile(pattern = "folder"))
dir.create(path)
write_fst(iris, file.path(path, "1.fst"))
list.files(path)
#> [1] "iris.fst"
df = disk.frame::disk.frame(path)
df
#> path: "/tmp/RtmpUzANvL/folder7f68686c046f"
#> nchunks: 1
#> nrow: 150
#> ncol: 5
df %>% 
  collect()
```
the above works and also

```r
library(fst)
library(disk.frame)
setup_disk.frame() # for multi worker processing 

path <- paste0(tempfile(pattern = "folder"))
df = disk.frame(path)
add_chunk(df, iris)
df
#> path: "/tmp/RtmpUzANvL/folder7f68686c046f"
#> nchunks: 1
#> nrow: 150
df %>% 
  collect()
```

**Comment by kendonB** [comment]
I see. Perhaps you could add an `order` argument to `disk.frame` to handle cases like this? My real life example has many chunks (not named with `1.fst` etc) already there in the folder.

You could also add a more useful error message.

This case was also extra confusing as the first step worked without error (you can see the number of rows in the `disk.frame`).

**Comment by xiaodaigh** [comment]
Thanks kendon. I will prioritise this. Thank you for taking the time to test it and report bugs.

**Comment by kendonB** [comment]
Thanks for the quick response!

**Comment by xiaodaigh** [comment]
That's cos I am in Melbourne :)

**Comment by xiaodaigh** [comment]
Fixed by #139. See
```r
library(fst)
library(disk.frame)
path <- paste0(tempfile(pattern = "folder"))
dir.create(path)
write_fst(iris, file.path(path, "iris.fst"))
write_fst(iris, file.path(path, "iris-another.fst"))
list.files(path)
#> [1] "iris.fst"
df = disk.frame::disk.frame(path)
df
#> path: "/tmp/RtmpUzANvL/folder7f68686c046f"
#> nchunks: 2
#> nrow: 300
#> ncol: 5
df %>% 
  collect()
```

**Comment by xiaodaigh** [comment]
Please reopen or submit a new issue if it doesn't work for your use-case.
